### PR TITLE
Cross mio updates

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Adding new sensors to sensor bulk load:

- 2 RM Young WND sensors (different UID 'METRMY' agreed upon by RT/SW due to there already being SWND sensors w/ same 3-digit SN using the standard CGINS-METWND-xxxxx format)
- 3 SeapHOx sensors (1 PHSEN-G, 2 PHSEN-H)
- 1 new OPTAA-D

Added SeapHOx PHSEN-G and -H to sensor_vocab

Asset Tracking can be used to verify, as well as the 2 attached files showing what I added.

[sensor_bulk_load-AssetRecord_2024-05-15.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/15324795/sensor_bulk_load-AssetRecord_2024-05-15.xlsx)
[sensor_vocab_2024-05-15.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/15324796/sensor_vocab_2024-05-15.xlsx)

### CODE REVIEW

**Comment by rtravis-whoi** [comment]
[Irminger 11 AM Updates](https://trello.com/c/jAVA6WUL/237-irminger-11-am-updates)

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1876,6 +1876,8 @@ CGINS-METPRC-00271,A02007,Sensor,0,Bulk Meteorology Instrument Package: METBK PR
 CGINS-METPRC-00272,A01722,Sensor,0,Bulk Meteorology Instrument Package: METBK PRC Module,METBK PRC MODULE,Star Engineering,ABS103-PRC,PRC272,,20150815,8560,,
 CGINS-METPRC-00273,A01724,Sensor,0,Bulk Meteorology Instrument Package: METBK PRC Module,METBK PRC MODULE,Star Engineering,ABS103-PRC,PRC273,,20150815,8560,,
 CGINS-METPRC-00276,A02137,Sensor,0,Bulk Meteorology Instrument Package: METBK PRC Module,METBK PRC MODULE,Star Engineering,ABS103-PRC,PRC276,,,,,Woods Hole Oceanographic Institution
+CGINS-METRMY-00244,,Sensor,0,Bulk Meteorology Instrument Package: METBK WND Module,METBK WND MODULE,R. M. Young,VO85010X-WND,WND244,,,,,Woods Hole Oceanographic Institution
+CGINS-METRMY-00245,,Sensor,0,Bulk Meteorology Instrument Package: METBK WND Module,METBK WND MODULE,R. M. Young,VO85010X-WND,WND245,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:1880]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 2]
```diff
@@ -2168,6 +2170,7 @@ CGINS-OPTAAD-00267,N00702,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIE
 CGINS-OPTAAD-00315,,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIES D,WET Labs,AC-S,ACS-315,,,,,
 CGINS-OPTAAD-00344,,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIES D,WET Labs,AC-S,ACS-344,,,,,
 CGINS-OPTAAD-00397,,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIES D,WET Labs,AC-S,ACS-397,,,,,
+CGINS-OPTAAD-00422,,Sensor,0,Spectrophotometer: OPTAA Series D,OPTAA SERIES D,WET Labs,AC-S,ACS-422,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:2173]: Looks good.

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 3]
```diff
@@ -2428,6 +2431,9 @@ CGINS-PHSENF-P0180,A01713,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Su
 CGINS-PHSENF-P0181,A01714,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0181,,20150818,18827.5,,
 CGINS-PHSENF-P0182,A01715,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0182,,20150818,18827.5,,
 CGINS-PHSENF-P0183,A01716,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0183,,20150818,18827.5,,
+CGINS-PHSENG-02054,,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,SeapHOx (350 m),721-2054,,,,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02064,,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,SeapHOx (2000 m),721-2064,,,,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02065,,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,SeapHOx (2000 m),721-2065,,,,,Woods Hole Oceanographic Institution
```
Comment (snwhite) [inline bulk/sensor_bulk_load-AssetRecord.csv:2436]: Looks good.  But should be "Deep SeapHOx".

File: bulk/sensor_vocab.csv [hunk 4]
```diff
@@ -120,6 +120,8 @@ Seawater pH: PHSEN Series A,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series D,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series E,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series F,Sunburst Sensors,SAMI-pH
+Seawater pH: PHSEN Series G,Sea-Bird Electronics,SeapHOx (350 m)
+Seawater pH: PHSEN Series H,Sea-Bird Electronics,SeapHOx (2000 m)
```
Comment (snwhite) [inline bulk/sensor_vocab.csv:124]: Looks good.  But should be "Deep SeapHOx".

**Review by snwhite — CHANGES_REQUESTED** [review]
The new PHSEN is the Deep SeapHOx.

File: bulk/sensor_vocab.csv [hunk 5]
```diff
@@ -120,6 +120,8 @@ Seawater pH: PHSEN Series A,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series D,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series E,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series F,Sunburst Sensors,SAMI-pH
+Seawater pH: PHSEN Series G,Sea-Bird Electronics,Deep SeapHOx (350 m)
+Seawater pH: PHSEN Series H,Sea-Bird Electronics,Deep SeapHOx (2000 m)
```
Comment (cooleyky) [inline bulk/sensor_vocab.csv:124]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv [hunk 6]
```diff
@@ -2428,6 +2431,9 @@ CGINS-PHSENF-P0180,A01713,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Su
 CGINS-PHSENF-P0181,A01714,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0181,,20150818,18827.5,,
 CGINS-PHSENF-P0182,A01715,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0182,,20150818,18827.5,,
 CGINS-PHSENF-P0183,A01716,Sensor,0,Seawater pH: PHSEN Series F,PHSEN SERIES F,Sunburst Sensors,SAMI-pH,P0183,,20150818,18827.5,,
+CGINS-PHSENG-02054,,Sensor,0,Seawater pH: PHSEN Series G,PHSEN SERIES G,Sea-Bird Electronics,Deep SeapHOx (350 m),721-2054,,,,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02064,,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,Deep SeapHOx (2000 m),721-2064,,,,,Woods Hole Oceanographic Institution
+CGINS-PHSENH-02065,,Sensor,0,Seawater pH: PHSEN Series H,PHSEN SERIES H,Sea-Bird Electronics,Deep SeapHOx (2000 m),721-2065,,,,,Woods Hole Oceanographic Institution
```
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:2436]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 2 above)
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:2173]: looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (cooleyky) [inline bulk/sensor_bulk_load-AssetRecord.csv:1880]: looks good

**Review by cooleyky — APPROVED** [review]
looks good

File: bulk/sensor_bulk_load-AssetRecord.csv (hunk 6 above)
Comment (snwhite) [reply bulk/sensor_bulk_load-AssetRecord.csv:2436]: Looks good.

File: bulk/sensor_vocab.csv (hunk 5 above)
Comment (snwhite) [reply bulk/sensor_vocab.csv:124]: Looks good.

**Review by snwhite — APPROVED** [review]

**Pull request merged**